Sources/Positionless/Algorithms/Rotate.swift

```diff
@@ -1,4 +1,4 @@
-extension MutableCollectionPartition {
+extension MutablePartition {
```

Comment:
```suggestion
extension MutablePartitioning {
```

Seems more appropriate.  It's not a single partition; it's a partitioning of some collection.